### ISSUE TITLE
Add support to unicode in file names

### ISSUE THREAD

**Issue by regystro** [idrassi/DirHash#6]
Hi. I'm using last version (1.12.0), and I've noticed that some file names containing unicode characters are wrongly encoded and fail during verify.

Command line:
`dirhash "testdir" -progress -sum -t testdir.blk3 -overwrite`

E.g.: a file called "Music001 – Live.mp3" which contains an unicode ["En dash" (0x2013)](https://www.online-toolz.com/tools/character-map-unicode-symbol.php?chid=2013)
is encoded as "Music001 ? Live.mp3" and then the verify fails:
```
Using Blake3 to verify hash of "testdir" ...
Error: file "testdir\Music001 ? Live.mp3" not found in checksum file.
```
The same happens to this file name: "Půjdem spolu do Betléma - Czech Children's Songs.mp3" which uses [this](https://www.online-toolz.com/tools/character-map-unicode-symbol.php?chid=016F) symbol.

Could you please add support for unicode characters?

Thank you.

**Comment by idrassi** [comment]
Thank you for reporting this issue.

The problem is caused by the fact that the checksum file created by DirHash was using ASCII encoding and so it could not store UNICODE file names and paths correctly.
I'm in the process of fixing this but using UTF-8 encoding for the checksum file.
I will also fix th display of UNICODE characters in Windows console because Windows console is not compatible with UNICODE by defaut.

**Comment by idrassi** [comment]
I have published version 1.13 that fixes this issue.
Please feel free to reopen this issue if you still have problems.

**Comment by regystro** [comment]
Everything working flawlessly with 1.13. Thanks for the quick fix!
